README.md

```diff
@@ -25,6 +25,14 @@ FetchContent_Declare( freertos_kernel
 )
 ```
 
+In case you prefer to add it as a git submodule, do:
+
+```bash
+$ git submodule add https://github.com/FreeRTOS/FreeRTOS-Kernel.git <path of the submodule>
+$ git submodule update --init
+$ git add submodule && git commit -m "Changed submodule branch to ..." && git push
```

Comment:
Is submodule the path of the submodule?
```suggestion
$ git add <path of the submodule> && git commit -m "Changed submodule branch to ..." && git push
```

Reply:
honestly, that's a redundant command. The idea was to give the user the idea to go inside the submodule path, checkout to a branch, exit, and do a `git add` again. 

It can just be removed, I'll do it, thanks/